### PR TITLE
CCBER updating progress report 2

### PR DESCRIPTION
**Pull request by angelchen7**

### CODE REVIEW

File: ucsb_ds_capstone_projects_2021/projects/ccber/description.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # ccber
-Project description
+![](https://github.com/angelchen7/ucsb-ds-capstone-2021.github.io/blob/main/ucsb_ds_capstone_projects_2021/projects/ccber/CCBER_logo.jpg?raw=true)
```
Comment (syoh) [inline ucsb_ds_capstone_projects_2021/projects/ccber/description.md]: Markdown can embed images directly from the same directory: https://jupyterbook.org/content/figures.html
For the image  `CCBER_logo.jpg` in the same directory (you already have it), replace the HTML with `some-text`

File: ucsb_ds_capstone_projects_2021/projects/ccber/update1.md [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+
+## Progress
+### Week 1
 We spent the first week learning about our research interest ([bees](https://www.wikiwand.com/en/Bee)) and exploring the [Global Biotic Interactions (GloBI)](https://www.globalbioticinteractions.org/about) dataset. Our data come from literature reports, human observations, and natural history specimen. 
-Goal of GloBI is to structure data into a usable form (csv., tsv.). \
-<img src="http://clipart-library.com/images/yTk4eXgTE.png" width="200" />\
-During our weekly meeting with the sponsors, we gained substantial knowledge about bees, especially the vital role it plays in global biotic interactions. In particular, we want to focus our research on the interactions between bees and plants. Interactions are directional, i.e. "Apis Mellifera pollinates brasicca rapa" will be categorized into source, interaction, and target, respectively.\
+Goal of GloBI is to structure data into a usable form (csv., tsv.). 
+
+![](https://github.com/angelchen7/ucsb-ds-capstone-2021.github.io/blob/main/ucsb_ds_capstone_projects_2021/projects/ccber/bee.PNG?raw=true)
```
Comment (syoh) [inline ucsb_ds_capstone_projects_2021/projects/ccber/update1.md]: Same here
Markdown can embed images directly from the same directory: https://jupyterbook.org/content/figures.html
For the image  `some-image.png` in the same directory (you already have it), replace the HTML with `some-text`

File: ucsb_ds_capstone_projects_2021/projects/ccber/update1.md [hunk 3]
```diff
@@ -17,24 +26,41 @@ Then, we discussed our results from using rglobi, the GloBI Web API, and Cypher.
 
 The goals after this meeting were to take some time to explore the raw dataset, since it was huge, check for reflexiveness in the data, and begin thinking about how we want to visualize these pollination networks. Additionally, we needed to brainstorm how to work with large data files that need to be shared and updated periodically. Some of us chose to explore the dataset with R while others were more comfortable with Python.
 
-## Week 3
+### Week 3
 We discussed the attributes and qualities of the raw data. Also, we explored different options for querying and subsetting the data in more detail. Our sponsors provided more insight on variations in the interactions data. Some of this variation includes: size, coloration, foraging behavior, and sociality of bees. There is significant diversity in the Apidae bee family. Classification and taxonomy can be used to filter some of this variation. 
 
 ![](https://github.com/angelchen7/ucsb-ds-capstone-2021.github.io/blob/main/ucsb_ds_capstone_projects_2021/projects/ccber/different_bees.png?raw=true "variation in Apidae")
```
Comment (syoh) [inline ucsb_ds_capstone_projects_2021/projects/ccber/update1.md]: Same here
Markdown can embed images directly from the same directory: https://jupyterbook.org/content/figures.html
For the image  `some-image.png` in the same directory (you already have it), replace the HTML with `some-text`

**Review by syoh — CHANGES_REQUESTED** [review]

**Comment by angelchen7** [comment]
Hi Professor, I believe I have made the appropriate changes in description.md and update1.md. Please let me know if there are any problems or additional edits to be made.

**Pull request merged**